### PR TITLE
aws: new resources

### PR DESCRIPTION
**Pull request by talset**
Implement new AWS resources

  * aws_db_subnet_group
  * aws_key_pair
  * aws_vpc_peering_connection
  * aws_alb_target_group
  * aws_alb_listener
  * aws_alb_listener_rule

### CODE REVIEW

File: aws/cmd/functions.go [hunk 1]
```diff
@@ -34,6 +34,16 @@ var (
 			// Returned values are commented in the interface doc comment block.
 			`,
 		},
+		Function{
+			Entity:  "VpcPeeringConnections",
```
Comment (xescugc) [inline aws/cmd/functions.go]: Should it maybe be named `VPC..`?

File: aws/resources.go [hunk 2]
```diff
@@ -427,6 +436,131 @@ func albs(ctx context.Context, a *aws, resourceType string, tags []tag.Tag) ([]p
...
+
+			if err != nil {
+				return nil, err
+			}
+
+			resources = append(resources, r)
+		}
+	}
+
+	//TODO: This resource it's not Importable yet (https://www.terraform.io/docs/providers/aws/r/lb_listener_certificate.html)
```
Comment (xescugc) [inline aws/resources.go]: I would add an issue and link it to this so we know/remember this if at somepoint it's importable.

If not, maybe we do not need this function as it has no use hehe.

File: aws/cmd/functions.go (hunk 1 above)
Comment (talset) [reply aws/cmd/functions.go]: :+1: 

I took the format from https://github.com/cycloidio/terracognita/blob/master/aws/cmd/functions.go#L29

Should we change both to VPC ?

**Comment by talset** [comment]
PR updated

File: aws/cmd/functions.go (hunk 1 above)
Comment (xescugc) [reply aws/cmd/functions.go]: Ok nothing, this is the one used on their side, it's ok :).

**Comment by xescugc** [comment]
CI fails

**Comment by talset** [comment]
> CI fails https://console.cycloid.io/projects/cycloid-owl/terracognita/pipelines/terracognita-master/jobs/test-pr/builds/124781#tests

yes I'm reverting VPCPeeringConnections to VpcPeeringConnections but I failed somewhere :)
I'm on it

**Pull request merged**